src/streamlink/plugins/ntv.py

```diff
@@ -0,0 +1,21 @@
+import re
+from streamlink.plugin import Plugin
+from streamlink.stream import HLSStream
+
+
+class NTV(Plugin):
+    url_re = re.compile(r"https?://(?:www\.)?ntv.ru/air/$")
+
+    @classmethod
+    def can_handle_url(cls, url):
+        return cls.url_re.match(url) is not None
+
+    def _get_streams(self):
+        tns = 'http://www.ntv.ru/services/live/tns/'
+        body = self.session.http.get(tns).text
+        d = {}
+        exec(body, d)
```

Comment:
The use of `exec` is not permitted, it's very insecure to `exec` unvalidated input. Instead you could do 
```suggestion
        d = dict(re.findall(r'(\w+)=\'(.*?)\'', body))
```

Reply:
> The use of `exec` is not permitted, it's very insecure to `exec` unvalidated input. Instead you 

Already fixed. Thank you @beardypig